build/rules.gni

```diff
@@ -0,0 +1,55 @@
+import("//build/config/mac/mac_sdk.gni")
+
+# This is imported from /ios becuase this functionality was moved
+# after Chromium stopped using xib files for macOS menu functionality
+# See https://chromium-review.googlesource.com/c/chromium/src/+/1648695
+import("//build/config/ios/rules.gni")
+
+# Template to compile and package Mac XIB files as bundle data.
```

Comment:
Add a note that this was copied from chromium & link the CL in which it was removed